drivers/sensors/fs3000_uorb.c

```diff
@@ -0,0 +1,306 @@
+/****************************************************************************
+ * drivers/sensors/fs3000_uorb.c
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.  The
+ * ASF licenses this file to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance with the
+ * License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.  See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ *
+ ****************************************************************************/
+
+/****************************************************************************
+ * Included Files
+ ****************************************************************************/
+
+#include <nuttx/config.h>
+
+#include <stdio.h>
+#include <stdlib.h>
+#include <debug.h>
+
+#include <nuttx/nuttx.h>
+#include <nuttx/kmalloc.h>
+#include <nuttx/kthread.h>
+#include <nuttx/signal.h>
+#include <nuttx/fs/fs.h>
+#include <nuttx/i2c/i2c_master.h>
+#include <nuttx/sensors/sensor.h>
+#include <nuttx/sensors/fs3000.h>
+
+#if defined(CONFIG_I2C) && defined(CONFIG_SENSORS_FS3000)
+
+/****************************************************************************
+ * Pre-processor Definitions
+ ****************************************************************************/
+
+#define FS3000_ADDR           0x28
+
+/****************************************************************************
+ * Private Type Definitions
+ ****************************************************************************/
+
+/* This is struct is just for "aesthetics", not used */
+
+struct fs3000_sensor_s
+{
+  struct sensor_lowerhalf_s lower;  /* Common lower interface */
+};
+
+struct fs3000_dev_s
+{
+  struct fs3000_sensor_s dev;     /* Sensor private data */
```

Comment:
```suggestion
  struct sensor_lowerhalf_s lower;     /* Sensor private data */
```
and remove fs3000_sensor_s